antsibull/cli/antsibull_build.py

```diff
@@ -101,9 +106,13 @@ def _normalize_release_build_options(args: argparse.Namespace) -> None:
     if args.command not in ('single', 'multiple', 'rebuild-single'):
         return
 
+    version_prefix = (
+        f'-{args.ansible_version.major}' if args.ansible_version.major > 2
+        else f'-{args.ansible_version.major}.{args.ansible_version.minor}'
+    )
```

Comment:
I  think we need to rename either this variable or version_suffix.  OItherwise line 124 `version_suffix = f'-{version_prefix}'` doesn't make a lot of sense to the first time code reader.  Maybe `compatible_version` or `compat_version_part` or something similar.

Reply:
Oh.... looking at the build error for building the package, I think that:
* version_prefix and version_suffix might actually be the same things?  So rename this to version_suffix and don't set it later.
* When it's set here, the `-` separator is included.  So on use, we should not be adding the dash again.

Reply:
I also just noticed the `-`. I think I'll remove it here and call this variable `compat_version_part`.

Reply:
Adjusted in 24740f469e21949ef697a05e9bcb6317d406fcb7.